### ISSUE TITLE
Follow git-config conditional includes

### ISSUE THREAD

**Issue by metafeather** [ForkIssues/Tracker#674]
Using Fork.app v1.0.81, it does not appear that ```[include]``` and ```[includeIf]``` are followed from ~/.gitconfig, ref: https://git-scm.com/docs/git-config#_includes

I personally use this to setup identities on a per-repo basis rather than add them to a repo config itself since I can version my file.

**Comment by Stanzilla** [comment]
Yeah this is highly annoying, it works fine in the terminal and I only recently noticed that Fork ignores it

**Comment by Stanzilla** [comment]
@DanPristupov any reason or maybe hint for a workaround?

**Comment by DanPristupov** [comment]
I couldn't find any problem with the `[include]` and `[includeIf]` blocks, they work as expected.

**Comment by Stanzilla** [comment]
> I couldn't find any problem with the `[include]` and `[includeIf]` blocks, they work as expected.

Can you try with folders inside of a WSL container? So with paths like `\\wsl$\Ubuntu\home\stan\projects\work\xx\`

**Comment by DanPristupov** [comment]
@Stanzilla this is the Mac issue tracker.

> Can you try with folders inside of a WSL container? So with paths like `\\wsl$\Ubuntu\home\stan\projects\work\xx\`

It will work, if it works in git. Fork doesn't touch the git config in any way.

**Comment by Stanzilla** [comment]
> @Stanzilla this is the Mac issue tracker.

Oh oops.

**Comment by Stanzilla** [comment]
@DanPristupov Do you want me to open a new issue on the Windows tracker then? Because it does not work here. It does work on the command line, but not in Fork. It always uses whatever user was defined first in the gitconfig, even if it later gets changed in an include file.

**Comment by Stanzilla** [comment]
As mentioned here https://github.com/fork-dev/Tracker/issues/1248#issuecomment-882011636 the Settings UI will also not check for includes and always show the first defined name in the config
